### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Ihor-Borzov**
Changed action types in the user-reducer.ts

### CODE REVIEW

File: src/Components/Users/UsersContainer.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { connect } from "react-redux";
 import { getCurrentPage, getFollow_unfollowUserIds, getIsFetching, getPageSize, getUsersCount, getUsersSuperReselector } from "../../Redux/user-selector";
-import { setTermAC, setIsFriendAC, follow, getUsers, setCurrentPage, setTotalUsersCount, setUsers, toggleIsFetching, toggleIsFollowingProgress, unfollow } from "../../Redux/users-reducer";
+import { actions, follow, getUsers, unfollow } from "../../Redux/users-reducer";
```
Comment (jacobvr) [inline src/Components/Users/UsersContainer.tsx:4]: If you decided to move most of the things to `actions`, what is the reason behind leaving `follow` and `unfollow` not under `actions`? Just trying to understand your logic here...

Comment (Ihor-Borzov) [reply src/Components/Users/UsersContainer.tsx:4]: follow, unfollow, getUsers are thunk creators

Comment (jacobvr) [reply src/Components/Users/UsersContainer.tsx:4]: Makes sense. This PR is just about moving things around, so I don't really have anything to comment on much. Feel free to merge it.

Comment (Ihor-Borzov) [reply src/Components/Users/UsersContainer.tsx:4]: Thank you for the review anyway! 
For now I work less on features and more on moving the project from JS to TS, currently I more focused on mastering Typescript - that is why it seems like moving things around

**Pull request merged**